pycloudlib/cloud.py

```diff
@@ -285,3 +285,39 @@ def _check_and_set_config(
             self.config = {}
         else:
             self.config = parse_config(config_file)[self._type]
+
+    @staticmethod
+    def _validate_tag(tag: str):
```

Comment:
Do we need to return the tag? Why not simply it and not returning?

If we insist in returning the tag, could we fully type it, please?

```suggestion
    def _validate_tag(tag: str) -> str:
```

And the same in child methods.